src/cpp/src/utils.hpp

```diff
@@ -196,6 +205,24 @@ template<class... Ts> struct overloaded : Ts... {using Ts::operator()...;};
 template<class... Ts> overloaded(Ts...) -> overloaded<Ts...>;
 std::shared_ptr<StreamerBase> create_streamer(StreamerVariant streamer, Tokenizer tokenizer);
 
+std::optional<ov::Any> pop_option(ov::AnyMap& config, const std::string& option_name);
+
+template <typename T>
+T pop_or_default(ov::AnyMap& config, const std::string& key, const T& default_value) {
+    auto anyopt = pop_option(config, key);
+    if (anyopt.has_value()) {
+        if (anyopt.value().empty()) {
+            if (ov::genai::utils::is_container<T>)
```

Comment:
I would vote against such lifehacks..

Reply:
Oh yes, you're right, we've already discussed this. In fact, such handling isn't really needed any longer as it was used only for `STATELESS`. I'll drop it!

Reply:
Done